src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public enum Colors {

```

Comment:
Enum naming convention in singular
```suggestion
public enum Color {

```